libraries/wallet/wallet.cpp

```diff
@@ -79,163 +80,27 @@
 # include <sys/stat.h>
 #endif
 
+#include "wallet_account.cpp"
```

Comment:
Need to be careful here. If ``make`` compiles these as objects (i.e. cmake tells it to), linkers will not be able to link.

This is certainly a way to fix the issue reported, and the precompiler will do its job. But I wonder if this is the best way to fix it. Honestly, I would break these into separate classes (ACID principle), but that would require a larger effort. Perhaps @abitmore or @pmconrad  or @oxarbitrage has an idea on how it could be done.

Reply:
Thanks ! My solution requires minor changes just to separate different logical parts in different files like for example `database.cpp` implementation.

Reply:
ok sure ! Thanks !